src/app/pages/interest-form/interest-form.hbs

```diff
@@ -0,0 +1,203 @@
+            Introductory Statistics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Prealgebra">
+            Prealgebra
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Precalculus">
+            Precalculus
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Economics">
+            Principles of Economics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Macroeconomics">
+            Principles of Macroeconomics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Macroeconomics for AP® Courses">
+            Principles of Macroeconomics for AP® Courses
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Microeconomics">
+            Principles of Microeconomics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Microeconomics for AP® Courses">
+            Principles of Microeconomics for AP® Courses
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Psychology">
+            Psychology
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="U.S. History">
+            U.S. History
+        </label>
+    </fieldset>
+    <fieldset class="labeled-inputs">
+        <label>
+            <span>Students per semester</span>
+            <input name="adoption_interest[students_per_semester]" type="number" required="true">
+        </label>
+    </fieldset>
+    <div class="section">
+    <p>
+        Which partners may we share your information with so they may contact you about
+        additional resources supporting the OpenStax College books?
+    </p>
+    <fieldset class="labeled-buttons">
+        <label>
+            <input name="adoption_interest[partner_resource][]" type="checkbox" value="Online homework providers">
+            Online homework providers
+        </label>
+        <label>
+            <input name="adoption_interest[partner_resource][]" type="checkbox" value="Courseware providers">
+            Courseware providers
+        </label>
+        <label>
+            <input name="adoption_interest[partner_resource][]" type="checkbox" value="Print custom editions">
+            Print custom editions
+        </label>
+    </fieldset>
+    <p class="">How did you hear about OpenStax College?
```

Comment:
Where's the closing tag for this element?  And it has an empty class, which should just be deleted.